app/views/admin/stores/index.html.erb

```diff
@@ -0,0 +1,39 @@
+
+<div class="container">
+  <h3>All Stores</h3>
+  <nav class="nav nav-tabs" id="myTab" role="tablist">
+    <a class="nav-item nav-link active" id="nav-home-tab" data-toggle="tab" href="#nav-home" role="tab" aria-controls="nav-home" aria-expanded="true">Active</a>
+    <%= link_to "Pending", admin_stores_pending_path, class: "nav-item nav-link" %>
+    <%= link_to "Suspended", admin_stores_suspended_path, class: "nav-item nav-link" %>
+  </div>
```

Comment:
All of these views are the same except for lines 4-8, theres a way to make this section dynamic so that we don't need three separate views.